docs/index.asciidoc

```diff
@@ -16,58 +12,31 @@ data from sources like logs and metrics or IT operations
 analytics and security analytics.
 
 ECS is still under development and backward compatibility is not guaranteed. Any
-feedback on the general structure, missing fields, or existing fields is appreciated.
-For contributions please read the  https://github.com/elastic/ecs/blob/master/CONTRIBUTING.md[Contribution Guidelines].
+feedback on the general structure, missing fields, or existing fields is
+appreciated. For contributions please read the
```

Comment:
I think for 1.0, we should change the message here.

Here's what we're now shooting for:

- We are following Semantic Versioning, meaning:
  - Backwards compatibility during the 1.x lifetime
  - Possibility of breaking changes only at major versions
- Related: We will align ECS major version with Elastic Stack major versions

At least that's how I think we should approach it. This stability will be a big decision factor for people deciding whether or not to adopt ECS.

WDYT @MikePaquette?

cc @ruflin

Reply:
Also while not technically a beta anymore, perhaps we should still guard and say "this is the first official version, so consider accordingly".

Reply:
@webmat I like the message you propose.  I would make sure to position these as goals, as there may be unforeseen situations that cause us to change these plans.

I would skip the "first official version" note - I the 1.0.0 version already makes that clear.